packages/frontend/src/utils/date.ts

```diff
@@ -5,6 +5,7 @@ import jstz from 'jstz';
 export const DATE_FORMAT = 'yyyy-MM-dd';
 export const DATE_FORMAT_TZ = 'yyyy-MM-dd z';
 export const DATE_FORMAT_LONG = 'yyyy-MM-dd HH:mm:ss';
+export const DATE_FORMAT_LONG_NAME = 'dd MMM yyyy, HH:mm';
```

Comment:
The date format shown in the tooltip should include the day-of-week name, see mockup:
![image](https://user-images.githubusercontent.com/159270/170280910-71cf423d-389b-413f-90e1-0f28c7b787dc.png)